Roles/Impostor/Swooper.cs

```diff
@@ -175,7 +174,6 @@ public static string GetHudText(PlayerControl pc)
 
     public static bool OnCheckMurder(PlayerControl killer, PlayerControl target)
     {
-        if (Medic.ProtectList.Contains(target.PlayerId)) return true;

```

Comment:
Translation removes some code

Reply:
Okay, I'll fix it

Reply:
LezaiYa and I will continue to be responsible for translating Simplified Chinese and Traditional Chinese